src/index.html

```diff
@@ -9,6 +12,23 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <img src="images/logo.png" alt="logo">
+      <nav>
+        <ul>
+          <li><a href="Apple" class="is-active">APPLE</a> </li>
```

Comment:
1. add classes to all tags
2. write link text using lowercase and change this case at css
3. use a correct link at `href`. If you don't have a link, you can use just `#`
4. apply code styling recommendations from a [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)
5. apply those recommendation all over the page

```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li>
            <a href="#apple" class="nav__list is-active">
              apple
            </a> 
          </li>
```